airbyte-integrations/bases/connector-acceptance-test/tools/acceptance_test_config_migration/create_issues.py

```diff
@@ -0,0 +1,113 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+
+import argparse
+import json
+import logging
+import os
+import subprocess
+import tempfile
+from typing import Any, Text, Dict, Optional
+
+import definitions
+import utils
+from jinja2 import Environment, FileSystemLoader
+
+## Update this line before running the script
+from migrations.strictness_level_migration import config
```

Comment:
This should also be able to be done automatically in the future. We could use [importlib](https://docs.python.org/3.9/library/importlib.html#importlib.import_module) to import a variable module name that we provide via the CLI or via an env var, for example